### ISSUE TITLE
Implement Google Analytics tracking

### ISSUE THREAD

**Issue by reshamas** [data-umbrella/event-board-web#57]
@jtorreggiani 
Can we add this info to home page for tracking?

```
<!-- Google tag (gtag.js) -->
<script async src="https://www.googletagmanager.com/gtag/js?id=G-NP82011Y5G"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'G-NP82011Y5G');
</script>
```

**Comment by jtorreggiani** [comment]
@reshamas, yes we should be able to add Google tracking. There may be some consideration when embedding Google Analytics trackings in React. We may want to look into this package to implement GA tracking. https://github.com/react-ga/react-ga

**Comment by YolineB** [comment]
@jtorreggiani and @reshamas I have implemented the GA tracking package (PR #130 ) . Please let me know if there are further steps you would like me to take.
